target-chains/near/receiver/src/lib.rs

```diff
@@ -162,11 +163,12 @@ impl Pyth {
         Ok(())
     }
 
-    /// Return the deposit required to update a price feed.
-    pub fn get_update_fee_estimate(&self) -> u64 {
+    /// Return the deposit required to update a price feed. This is the upper limit for an update
+    /// call and any remaining deposit not consumed for storage will be refunded.
+    pub fn get_update_fee_estimate(&self) -> U128 {
```

Comment:
why did this need to change to U128?

(I think you should use whatever number representation NEAR typically uses for token quantities.)

Reply:
it's funny how everyone patches u128 for serde_json. i wish they could natively solve it.